diff --git a/ios/Extensions/Converters/AV+PixelFormat.swift b/ios/Extensions/Converters/AV+PixelFormat.swift
index f0c61086a44e165c3643ce8bd52b8fda62624ae5..d03b48794bb0bc62baa826ea198dd1f757d41424 100644
--- a/ios/Extensions/Converters/AV+PixelFormat.swift
+++ b/ios/Extensions/Converters/AV+PixelFormat.swift
@@ -48,10 +48,12 @@ extension PixelFormat {
       self = .rgbRgba8Bit
     case .rgbRgb8Bit:
       self = .rgbRgb8Bit
+#if arch(arm64)
     case .rawBayerPacked9612Bit:
       self = .rawBayerPacked9612Bit
     case .rawBayerUnpacked16Bit:
       self = .rawBayerUnpacked16Bit
+#endif
     default:
       let stringified = Self.fourCharCodeToString(mediaSubType.rawValue)
       logger.error("Unknown Format PixelFormat: \"\(stringified)\"")
