diff --git a/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift b/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift
index dc40d8ccce69b959d9f4ed1ff28112b30661e612..07643659273eee214e9ae65bc8c87327029876cb 100644
--- a/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift	
+++ b/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift	
@@ -25,7 +25,7 @@ extension ResolutionBiasConstraint: ResolvableConstraint {
     case .closestTo(let targetResolution):
       let targetResolution = targetResolution.toCMVideoDimensions()
       guard let closestSupportedResolution = supportedResolutions.nearest(to: targetResolution) else {
-        throw RuntimeError("Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
+        throw RuntimeError.error(withMessage: "Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
       }
       penalty = closestSupportedResolution.penalty(targetResolution)
     case .min(let minimumResolution):
@@ -37,7 +37,7 @@ extension ResolutionBiasConstraint: ResolvableConstraint {
         penalty = 0.0
       } else {
         guard let best = supportedResolutions.nearest(to: minimumResolution) else {
-          throw RuntimeError("Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
+          throw RuntimeError.error(withMessage: "Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
         }
         penalty = best.penalty(minimumResolution)
       }
