diff --git a/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package_header_only/prefab_publication.json/debug b/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package_header_only/prefab_publication.json/debug
new file mode 100644
index 0000000..d19e2d9
--- /dev/null
+++ b/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package_header_only/prefab_publication.json/debug
@@ -0,0 +1,17 @@
+{
+  "installationFolder": "/Users/devthang/Desktop/thangdev/rn-media-toolkit/react-native-media-toolkit/example/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package/debug/prefab",
+  "gradlePath": ":react-native-vision-camera",
+  "packageInfo": {
+    "packageName": "react-native-vision-camera",
+    "packageSchemaVersion": 2,
+    "packageDependencies": [],
+    "modules": [
+      {
+        "moduleName": "VisionCamera",
+        "moduleHeaders": "/Users/devthang/Desktop/thangdev/rn-media-toolkit/react-native-media-toolkit/example/node_modules/react-native-vision-camera/android/build/headers/visioncamera",
+        "moduleExportLibraries": [],
+        "abis": []
+      }
+    ]
+  }
+}
\ No newline at end of file
diff --git a/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package_header_only/prefab_publication.json/release b/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package_header_only/prefab_publication.json/release
new file mode 100644
index 0000000..027d0de
--- /dev/null
+++ b/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package_header_only/prefab_publication.json/release
@@ -0,0 +1,17 @@
+{
+  "installationFolder": "/Users/devthang/Desktop/thangdev/rn-media-toolkit/react-native-media-toolkit/example/node_modules/react-native-vision-camera/android/build/intermediates/prefab_package/release/prefab",
+  "gradlePath": ":react-native-vision-camera",
+  "packageInfo": {
+    "packageName": "react-native-vision-camera",
+    "packageSchemaVersion": 2,
+    "packageDependencies": [],
+    "modules": [
+      {
+        "moduleName": "VisionCamera",
+        "moduleHeaders": "/Users/devthang/Desktop/thangdev/rn-media-toolkit/react-native-media-toolkit/example/node_modules/react-native-vision-camera/android/build/headers/visioncamera",
+        "moduleExportLibraries": [],
+        "abis": []
+      }
+    ]
+  }
+}
\ No newline at end of file
diff --git a/node_modules/react-native-vision-camera/ios/Extensions/URL+createTempURL.swift b/node_modules/react-native-vision-camera/ios/Extensions/URL+createTempURL.swift
index 528b267..f8e8a80 100644
--- a/node_modules/react-native-vision-camera/ios/Extensions/URL+createTempURL.swift
+++ b/node_modules/react-native-vision-camera/ios/Extensions/URL+createTempURL.swift
@@ -32,9 +32,8 @@ extension URL {
    * in the temporary directory using a random `UUID` as its name.
    */
   static func createTempURL(fileType: AVFileType) throws(TemporaryFileError) -> URL {
-    // AVFileType is a string-like type (usually MIME, like "video/mp4")
-    let mimeType = fileType.rawValue
-    guard let utFileType = UTType(mimeType: mimeType) else {
+    // AVFileType is a UTI string in Apple systems (e.g. "public.mpeg-4" or "com.apple.quicktime-movie")
+    guard let utFileType = UTType(fileType.rawValue) else {
       throw TemporaryFileError.invalidAVFileType
     }
     return try createTempURL(fileType: utFileType)
diff --git a/node_modules/react-native-vision-camera/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift b/node_modules/react-native-vision-camera/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift
index dc40d8c..0764365 100644
--- a/node_modules/react-native-vision-camera/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift	
+++ b/node_modules/react-native-vision-camera/ios/Hybrid Objects/Constraints/ResolvableConstraint/ResolvableConstraint+ResolutionBiasConstraint.swift	
@@ -25,7 +25,7 @@ extension ResolutionBiasConstraint: ResolvableConstraint {
     case .closestTo(let targetResolution):
       let targetResolution = targetResolution.toCMVideoDimensions()
       guard let closestSupportedResolution = supportedResolutions.nearest(to: targetResolution) else {
-        throw RuntimeError("Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
+        throw RuntimeError.error(withMessage: "Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
       }
       penalty = closestSupportedResolution.penalty(targetResolution)
     case .min(let minimumResolution):
@@ -37,7 +37,7 @@ extension ResolutionBiasConstraint: ResolvableConstraint {
         penalty = 0.0
       } else {
         guard let best = supportedResolutions.nearest(to: minimumResolution) else {
-          throw RuntimeError("Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
+          throw RuntimeError.error(withMessage: "Format \(format) does not have any available resolutions for stream type \"\(streamType)\"!")
         }
         penalty = best.penalty(minimumResolution)
       }
